pkg/kubelet/dockertools/manager.go

```diff
@@ -1563,7 +1586,7 @@ func appendToFile(filePath, stringToAppend string) error {
 }
 
 // createPodInfraContainer starts the pod infra container for a pod. Returns the docker container ID of the newly created container.
-func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubetypes.DockerID, error) {
+func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubetypes.DockerID, error, string) {
```

Comment:
Can you document what's the returning string is? Alternatively, how about just returning a whole "startContainerResult"?